library/include/ck/library/utility/check_err.hpp

```diff
@@ -23,72 +23,80 @@
 namespace ck {
 namespace utils {
 
-template <typename ComputeDataType>
+template <typename ComputeDataType, typename OutDataType>
 double get_relative_threshold()
 {
     using F8   = ck::f8_t;
     using F16  = ck::half_t;
     using BF16 = ck::bhalf_t;
     using F32  = float;
+    using I8   = int8_t;
+    using I32  = int32_t;
 
-    int mantissa = 0;
-    if constexpr(is_same_v<ComputeDataType, F8>)
+    static_assert(is_same_v<ComputeDataType, F8> || is_same_v<ComputeDataType, F16> ||
+                      is_same_v<ComputeDataType, BF16> || is_same_v<ComputeDataType, F32> ||
+                      is_same_v<ComputeDataType, I8> || is_same_v<ComputeDataType, I32>,
+                  "Warning: Unhandled ComputeDataType for setting up the relative threshold!");
+    int compute_mantissa = 0;
+    if constexpr(is_same_v<ComputeDataType, I8> || is_same_v<ComputeDataType, I32> ||
+                 is_same_v<ComputeDataType, int>)
```

Comment:
You use here `int` data type, whereas you don't pass it through above static assert. Either remove it from here, or add there.

Reply:
It may be int in some cases. Added int handling to the static assert.